cruisetrack/process/workflow_lines.py

```diff
@@ -241,52 +271,25 @@ def process_lines(layer_provider, laye_r, is_individual_trackline, is_accessory,
             arrays = np.transpose([new_lon_st, new_lon_sp, new_lat_st, new_lat_sp])
             df = pd.DataFrame(arrays)
 
-    # if individual cruise track line should be followed
-    if is_individual_trackline:
-        ind_track = 1
-        lon_series = np.zeros([len(df), 200])
-        lat_series = np.zeros([len(df), 200])
-        coun_t = 0
-        for fea_t in laye_r.getFeatures():
-            geom = fea_t.geometry().constGet()
-            for n in list(range(len(geom[0]))):
-                vertices_on_line = QgsPointXY(geom[0][n])
-                lon_series[coun_t, n] = vertices_on_line[0]
-                lat_series[coun_t, n] = vertices_on_line[1]
-            coun_t = coun_t + 1
-
-        lon = []
-        lat = []
-
-        for n in list(range(coun_t)):
-            lon = lon + list(filter(lambda num: num != 0, lon_series[n]))
-            lat = lat + list(filter(lambda num: num != 0, lat_series[n]))
-
-        arrays = [lon, lat]
-        df = pd.DataFrame(arrays)
-        df = df.T
-    else:
-        ind_track = 0
-
     # flip across (ex-NS)
     if flip_ns:
         df = df.iloc[::-1]
 
     # make just two culumns Lon and Lat
     if is_individual_trackline:
-        lon = df[0]
-        lat = df[1]
+        lon = df['x']
+        lat = df['y']
 
     else:
         lis_t2 = list(range(0, len(df.index) * 2))
         odds_idx2 = lis_t2[1::2]
         evens_idx2 = lis_t2[::2]
         lon = np.zeros(len(df.index) * 2)
-        lon[odds_idx2] = df[0]
-        lon[evens_idx2] = df[1]
+        lon[odds_idx2] = df['X_start']
```

Comment:
I had to use different indexes here, need to check if they are all correct.
Named indexes are better to understand